src/features/defi/providers/idle/components/IdleManager/Deposit/components/Deposit.tsx

```diff
@@ -54,22 +54,25 @@ export const Deposit: React.FC<DepositProps> = ({
   const marketData = useAppSelector(state => selectMarketDataById(state, assetId))
 
   // user info
-  const balance = useAppSelector(state => selectPortfolioCryptoBalanceByAssetId(state, { assetId }))
+  const balance = useAppSelector(state =>
+    selectPortfolioCryptoBalanceByFilter(state, { assetId, accountId }),
```

Comment:
may want to memo this boy too

Reply:
d713fea24e984f81494c805eef38b97019021f69